keyboards/naked64/keymaps/default/keymap.c

```diff
@@ -0,0 +1,157 @@
+      _______,  KC_TAB,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                  KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_MINS, KC_BSPC, \
+  //|--------+--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                 KC_F6, KC_MINS,  KC_EQL, KC_LBRC, KC_RBRC, KC_BSLS, _______, \
+  //         |--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,                KC_F12, KC_NUHS, KC_NUBS, _______, _______, _______, _______, \
+  //         |--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______,          _______,          _______, _______,               _______,          _______, _______, _______, _______, _______  \
+          //`-----------------------------------------------------------------------------------------------------------------------------------'
+  ),
+
+  [_ADJUST] = LAYOUT( /* Base */
+  //,--------------------------------------------------------------|             |--------------------------------------------------------------.
+   TG(_FLOCK),   RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                 KC_F6,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,  KC_F12, \
+  //|--------+--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+      _______, _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,               RGB_RST, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, _______, \
+  //|--------+--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,               RGB_TOG, RGB_MOD, XXXXXXX,LCA(KC_DEL),LALT(KC_PSCR),KC_PSCR,XXXXXXX, \
+  //         |--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,               RGB_VAD, RGB_VAI, RGB_HUD, RGB_HUI, RGB_SAD, RGB_SAI, XXXXXXX, \
+  //         |--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______,          _______,          _______, _______,               _______,          _______, _______, _______, _______, _______  \
+          //`-----------------------------------------------------------------------------------------------------------------------------------'
+  )
+};
+
+
+//A description for expressing the layer position in LED mode.
+uint32_t layer_state_set_user(uint32_t state) {
```

Comment:
The layer state stuff has been typedef'd so that it isn't always uint32.  If you would, update this: 

```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
```